### DISCUSSION TITLE
Please take a look at pending review of project GMT

### DISCUSSION THREAD

**Discussion by Astat67** [WalletConnect/walletconnect-monorepo#2139] [Developer Support]
Hello to the WalletConnect team,

I was wondering if there is any feedback about our GMT placement app in the WalletConnect explorer? We expect verification in your platform within 3 weeks. Thanks!
 uuid=3d31d54f-8284-444b-af5e-0fe0e055d626
Project ID  55fb90434d51a5053a08287736dbb52a

**Comment by finessevanes** [comment]
Hey - we're actually unable to search via project ID or uuid. We are prioritizing wallets and dapps that have v2 integrations at the moment.
